src/app/components/wollo-input/index.js

```diff
@@ -26,6 +26,7 @@ export class WolloInput extends Component {
 
     setExchangedValue = (amount, currentCurrency) => {
         const {exchange, baseCurrency, onChangeAmount} = this.props;
+        amount = amount ? amount.replace(',', '') : amount;
```

Comment:
Needs regex with g flag to remove more than first comma

```suggestion
        amount = amount ? amount.replace(/,/g, '') : amount;
```